students/aravichander/lesson09/Mailroom_class.py

```diff
@@ -0,0 +1,146 @@
+#Mailroom_OO
+
+import sys
+import json
+import unittest 
+
+class Donor:
+	def __init__(self,Name):
+		self.Name = Name
+
+	@property
+	def donations(self):
+		return self.donations
+
+	@classmethod
+	def sumdonation(cls, donations):
```

Comment:
you don't want a classmethod here -- the sum of donations is a simple number that should be returned. 

a property is good in this case.

classmethods are for creating a new instance of a class for some reason -- no need for that here.